<ui version="4.0" >
 <class>SoftwareDB</class>
 <widget class="QDialog" name="SoftwareDB" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QStackedWidget" name="stackedPages" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>443</width>
        <height>353</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Name</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="nameinfoLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>not specified</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changeNamePushButton" >
             <property name="text" >
              <string>Change...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Compagny</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="compagnyComboBox" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Genre</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="genreComboBox" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Extentions</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="extentionsinfoLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>not specified</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="extentionsChangePushButton" >
             <property name="text" >
              <string>Change...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Machine</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QComboBox" name="machineComboBox" />
         </item>
         <item row="5" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Patched</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <widget class="QComboBox" name="patchedComboBox" />
         </item>
         <item row="6" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Type</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1" >
          <widget class="QComboBox" name="typeComboBox" />
         </item>
         <item row="7" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Year</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1" >
          <widget class="QComboBox" name="yearComboBox" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Number of matches:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numbermatchesLabel" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="nextPushButton" >
           <property name="text" >
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>443</width>
        <height>353</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableWidget" name="gamelistView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="showDropIndicator" stdset="0" >
           <bool>false</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
          <property name="columnCount" >
           <number>1</number>
          </property>
          <column/>
         </widget>
         <widget class="QFrame" name="frame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="label_name" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>Name</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_compagny" >
             <property name="text" >
              <string>Compagny</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent" >
              <number>11</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_year" >
             <property name="text" >
              <string>Year</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent" >
              <number>11</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_machine" >
             <property name="font" >
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text" >
              <string>Machine</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="indent" >
              <number>11</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_genre" >
             <property name="font" >
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text" >
              <string>Genre</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="indent" >
              <number>11</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Slideshow" native="1" name="widget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>139</width>
               <height>173</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="previousPushButton" >
           <property name="text" >
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextPushButton_2" >
           <property name="text" >
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>443</width>
        <height>353</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QFrame" name="frame_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_9" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>Suggested configuration for</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="gamenameLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font" >
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text" >
               <string>name_of_the_game</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QCheckBox" name="machineCheckBox" >
              <property name="text" >
               <string>Machine</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="machineLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="extensionsCheckBox" >
              <property name="text" >
               <string>Extensions</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLabel" name="extensionsLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="diskaCheckBox" >
              <property name="text" >
               <string>Diska</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QLabel" name="diskaLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QCheckBox" name="diskbCheckBox" >
              <property name="text" >
               <string>Diskb</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QLabel" name="diskbLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QCheckBox" name="cartaCheckBox" >
              <property name="text" >
               <string>Carta</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="QLabel" name="cartaLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QCheckBox" name="cartbCheckBox" >
              <property name="text" >
               <string>Cartb</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1" >
             <widget class="QLabel" name="cartbLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>81</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="previousPushButton_2" >
           <property name="text" >
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyPushButton" >
           <property name="text" >
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Slideshow</class>
   <extends>QWidget</extends>
   <header>slideshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
